engine/language-server/src/test/scala/org/enso/languageserver/boot/resource/RepoInitializationSpec.scala

```diff
@@ -35,7 +35,8 @@ class RepoInitializationSpec
   val Timeout: FiniteDuration = 10.seconds.dilated
 
   override def afterAll(): Unit = {
-    TestKit.shutdownActorSystem(system)
+    TestKit.shutdownActorSystem(system, verifySystemShutdown = true)
```

Comment:
There is a `TestKit` support in Akka and calling `shutdownActorSystem` doesn't really shut it down, unless one `verifySystemShutdown`!? Good you find it.

Reply:
I _think_ it was mostly shutting it down. This ensures that it really does it so that I don't have to chase ghosts.